common/laws/00_economic_system.txt

```diff
@@ -14,20 +26,18 @@ law_traditionalism = {
 	on_activate = {
 	}
 	modifier = {
-		state_shopkeepers_investment_pool_efficiency_mult = -0.5
+		state_shopkeepers_investment_pool_efficiency_mult = -0.4
 		state_capitalists_investment_pool_efficiency_mult = -0.5
-		state_farmers_investment_pool_efficiency_mult = -0.5	
-		state_clergymen_investment_pool_efficiency_mult = -0.5	
-		state_aristocrats_investment_pool_efficiency_mult = -0.5	
+		state_farmers_investment_pool_efficiency_mult = -0.3
+		state_clergymen_investment_pool_efficiency_mult = -0.3
+		state_aristocrats_investment_pool_efficiency_mult = -0.35
 		state_bureaucracy_population_base_cost_factor_mult = -0.25
 		country_government_dividends_reinvestment_add = 0.25
 		state_tax_capacity_mult = -0.25
 		country_private_construction_allocation_mult = 0.25
-		building_nationalization_cost_mult = 1.0
+		building_nationalization_cost_mult = 1
 		building_nationalization_investment_return_add = 0.1
 		country_leverage_resistance_mult = 0.25
-
-		state_market_access_price_impact = -0.15
```

Comment:
So are we also going to add the market_access_price_impact to railroads, docks, and that technology? If so I would like that to be a separate pull request.

Reply:
I will add that as a separate pull request after 1.7.2 then